brainwallet/Bento Effects /ThemeGradientHelpers.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ThemeGradientHelpers.swift
+//  brainwallet
+//
+//  Created by Kerry Washington on 17/10/2025.
+//  Copyright © 2025 Grunt Software, LTD. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+ enum MainGradientStyle {
+    case darkStyle
+    case lightStyle
+
+    var colorThemeArray: [Color] {
+        switch self {
+        case .darkStyle:
+            return  [BrainwalletColor.surface.opacity(0.2), BrainwalletColor.grape.opacity(0.2), BrainwalletColor.midnight.opacity(0.1)]
+        case .lightStyle:
+            return [BrainwalletColor.surface.opacity(0.1), BrainwalletColor.grape.opacity(0.1), BrainwalletColor.midnight.opacity(0.1)]
+        }
+    }
+
+     var maskGradientStops: [Color] {
```

Comment:
This should be platform common @JosephSanjaya . It sets the mask for the bento background animation. You should be able to copy almost directly. Or , if we like and update we both change it.  This is so the toolbar on the bottom is always the surface color theme